R/estimate_secondary.R

```diff
@@ -668,7 +668,7 @@ forecast_secondary <- function(estimate,
   )
   stan_data$n <- nrow(stan_data$primary)
   stan_data$t <- ncol(stan_data$primary)
-  stan_data$h <- nrow(primary[sample == min(sample)])
+  stan_data$horizon <- nrow(primary[sample == min(sample)])
```

Comment:
I think this one is actually called `h` in https://github.com/epiforecasts/EpiNow2/blob/fdeac6a54202eee6219f3917e80f63157bc7721e/inst/stan/simulate_secondary.stan#L14 so would have to change it in the stan model, too

Reply:
Thanks. Fixed in 59dec8649db78f00ef3bf8b8605f3f2d5a40ff53.